src/main/java/net/atos/zac/signalering/SignaleringenJob.java

```diff
@@ -213,7 +213,7 @@ private ZoekParameters getZaakSignaleringLaterTeVerzendenZoekParameters(final Da
             final Zaaktype zaaktype, final int venster) {
         final LocalDate now = LocalDate.now();
         final ZoekParameters parameters = getOpenZaakMetBehandelaarZoekParameters(zaaktype);
-        parameters.addDatum(veld, new DatumRange(now.plusDays(venster + 1), null));
+        parameters.addDatum(veld, new DatumRange(now.plusDays((long) venster + 1), null));
```

Comment:
Could use implicit conversion when using explicit long value 1L

Reply:
You mean `venster + 1L` instead? also possible but I thought this was a bit clearer. Both are valid according to http://localhost:9000/coding_rules?open=java%3AS2184&rule_key=java%3AS2184 so it's more a matter of taste.